src/jquery.addressfield.js

```diff
@@ -4,9 +4,13 @@
  *
  * Licensed under the MIT license.
  */
-
-(function ($) {
-
+(function(factory) {
+  if (typeof module === "object" && typeof module.exports === "object") {
+    factory(require("jquery"));
+  } else {
+    factory(jQuery);
+  }
+}(function factory($) {
```

Comment:
For the UMD snippet, check [here](http://blog.npmjs.org/post/112712169830/making-your-jquery-plugin-work-better-with-npm).